libraries/Microsoft.Bot.Builder.Azure/AzureBlobStorage.cs

```diff
@@ -28,6 +28,7 @@ namespace Microsoft.Bot.Builder.Azure
     /// property value to the blob's ETag upon read. Afterward, an <see cref="AccessCondition"/> with the ETag value
     /// will be generated during Write. New entities start with a null ETag.
     /// </remarks>
+    [Obsolete("This class is deprecated. Please use Microsoft.Bot.Builder.Azure.BlobsStorage instead.", false)]
```

Comment:
>deprecated [](start = 29, length = 10)

This isn't deprecated if you are still using the old SDK, no?  Perhaps we should make that clearer.

Reply:
Addressed